tour-of-go/exports/colors/colors_type.go

```diff
@@ -0,0 +1,13 @@
+package colors
+
+import "fmt"
+
+var blue = "I love blue color"
```

Comment:
When creating a global variable make sure you add the `type` 

There's a lot of this. Please check all your packages and effect the changes.